pkg/services/notifications/smtp.go

```diff
@@ -94,6 +95,12 @@ func (sc *SmtpClient) buildEmail(ctx context.Context, msg *Message) *gomail.Mess
 	m.SetHeader("To", msg.To...)
 	m.SetHeader("Subject", msg.Subject)
 
+	at := strings.LastIndex(msg.From, "@")
```

Comment:
this won't render correctly for a `From` value of something like: `"Foo Bar" <baz@example.com>`, which is quite a common use - we should probably also cover that case in the unit tests

Reply:
@hairyhenderson, could you take a look at this once more?